### DISCUSSION TITLE
How do you make the toc title diffrent from the actual title ?

### DISCUSSION THREAD

**Discussion by kwatman** [squidfunk/mkdocs-material#2535] [Q&A] [answered]
Hello everyone. I want to have a toc where the title of the link is different then the actual title on the page. I saw that it has been done in the docs of mkdocstring and I was wondering how you can do it. 
![help](https://user-images.githubusercontent.com/13818653/114167000-b2aa9080-992e-11eb-9a55-00c856a6a764.png)

**Comment by jaceklaskowski** [comment, marked as answer]
Duplicate of https://github.com/squidfunk/mkdocs-material/discussions/2500 ?

**Reply by kwatman** [reply]
Thank you. It worked
